worldedit-bukkit/build.gradle.kts

```diff
@@ -107,6 +107,8 @@ dependencies {
     implementation("org.bstats:bstats-base:2.2.1")
     compileOnlyApi("org.inventivetalent:mapmanager:1.7.10-SNAPSHOT") { isTransitive = false }
     implementation("com.github.TechFortress:GriefPrevention:16.17.1") { isTransitive = false }
+    implementation("com.github.bloodmc:GriefDefenderApi:920a610") { isTransitive = false }
+    implementation("com.flowpowered:flow-math:1.0.3") { isTransitive = false }
```

Comment:
What's the point of this library? The way you use it to create a vector is unneeded. WorldEdit itself provides tooling to create vectors, see Vector2/Vector3 for example.

Reply:
It's used by GriefDefender, if there aren't methods available to avoid it at all, there's no other way than using it.

Reply:
Oic, but, the way it's used here is to create a vector from 3 points which is possible without this lib.
If needed, expose it transitively thru the api if it's a hard dep for GD.

Reply:
flow-math is a hard dependence of GD.That means there would be a hard work if I don't use GD's API.What's more,That will make this hook not stable and not reliable.
Related Code in GD: https://github.com/bloodmc/GriefDefender/blob/master/bukkit/src/main/java/com/griefdefender/claim/GDClaim.java
Sorry,I have no time to slove it,you can have a look with this then make it to be a better implement.